pkg/minishift/addon/parser/addon_parser.go

```diff
@@ -182,10 +182,17 @@ func (parser *AddOnParser) parseCommands(scanner *bufio.Scanner) ([]command.Comm
 func createMetaData(header []string) (addon.AddOnMeta, error) {
 	regex, _ := regexp.Compile(`^# ?(.*):(.*)`)
 	metaMap := make(map[string]string)
+	var key string
 	for _, line := range header {
 		matches := regex.FindAllStringSubmatch(line, -1)
+		if len(matches) == 0 {
+			line = strings.TrimPrefix(line, commentChar)
+			metaMap[key] = metaMap[key] + " " + strings.TrimSpace(line)
+			continue
+		}
 		for _, match := range matches {
-			metaMap[strings.Trim(match[1], " ")] = strings.Trim(match[2], " ")
+			key = strings.Trim(match[1], " ")
```

Comment:
As mentioned in the other command, we might want to harden the regexp a bit, maybe:

    regexp.Compile(`^# ?([a-zA-Z-]*):(.*)`)

This way something like:

    # Description: This add-on
    # prints the host:port of the OpenShift cluster